requirements/tests.ini

```diff
@@ -1,5 +1,6 @@
 -r base.txt
 
+tox
```

Comment:
This means that we can run `tox` only if we install the requirements with `pip install -e tests.txt`. Is this expected?